pkgs/development/python-modules/black/default.nix

```diff
@@ -21,15 +24,17 @@
 
 buildPythonPackage rec {
   pname = "black";
-  version = "22.8.0";
+  version = "22.10.0";
 
-  disabled = pythonOlder "3.6";
+  disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-eS9+tUC6mhfoZWU4cB0+sa/LE047RbcfILJcd6jbfm4=";
+    hash = "sha256-9RNYjaWZlD4M3k4yzJh56CXVhyDWVXBi0QmMWtgAgOE=";
   };
 
+  format = "pyproject";
+
```

Comment:
```suggestion
```

Reply:
Done in https://github.com/NixOS/nixpkgs/pull/194899/commits/2b0bd6c93f83e483441c4a79d8980a7d2fe38105